src/app/+data-and-analytics/academic-age-group/academic-age-group.component.ts

```diff
@@ -0,0 +1,151 @@
+import { Component, EventEmitter, Input, OnChanges, OnInit, Output, QueryList, ViewChildren } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { Store, select } from '@ngrx/store';
+import { BehaviorSubject, Observable, Subject, filter, map, tap } from 'rxjs';
+
+import { SolrDocument } from '../../core/model/discovery';
+import { Filterable } from '../../core/model/request';
+import { DataAndAnalyticsView, DisplayView, OpKey } from '../../core/model/view';
+import { AppState } from '../../core/store';
+import { selectResourcesResearchAge } from '../../core/store/sdr';
+import { AcademicAge } from '../../core/store/sdr/sdr.reducer';
+import { fadeIn } from '../../shared/utilities/animation.utility';
+import { BarplotComponent, BarplotInput } from '../barplot/barplot.component';
+
+import * as fromSdr from '../../core/store/sdr/sdr.actions';
+
+const academicAgeGroupToBarplotInput = (academicAge: AcademicAge): BarplotInput => {
+  return {
+    label: academicAge.label,
+    data: academicAge.groups
+  } as BarplotInput;
+}
+
+const rk = 'Researchers';
+const pk = 'Publications';
+const apk = 'Average publications';
+
+@Component({
+  selector: 'scholars-academic-age-group',
+  templateUrl: './academic-age-group.component.html',
+  styleUrls: ['./academic-age-group.component.scss'],
+  animations: [fadeIn],
+})
+export class AcademicAgeGroupComponent implements OnInit, OnChanges {
+
+  @Input()
+  public organization: SolrDocument;
+
+  @Input()
+  public displayView: DisplayView;
+
+  @Input()
+  public dataAndAnalyticsView: DataAndAnalyticsView;
+
+  @Output()
+  public labelEvent: EventEmitter<string>;
+
+  @Input()
+  public upperLimitInYears = 40;
+
+  @Input()
+  public groupingIntervalInYears = 5;
+
+  @ViewChildren(BarplotComponent) barplots: QueryList<BarplotComponent>;
```

Comment:
Is there a reason to have this line structured differently?
This could also be `public`.
Is there a reason not to be `public`?

Example:
```ts
@ViewChildren(BarplotComponent)
public barplots: QueryList<BarplotComponent>;
```

Reply:
No reason. Will make it more consistent. There is a lot of cleanup and refactoring needed but will defer most for a dedicated PR.

Reply:
In Angular public is necessary for use in the template. Use of private for use only within the class and no modifier is public. 

https://www.typescriptlang.org/docs/handbook/2/classes.html#member-visibility
https://stackoverflow.com/questions/20696932/which-access-modifiers-are-implied-when-not-specified

Reply:
I see, so it is a styling practice.